src/v2/Components/v2/ArtworkFilter/ArtworkFilterContext.tsx

```diff
@@ -97,6 +98,9 @@ export interface ArtworkFilterContextProps {
   // Components
   ZeroState?: React.FC
 
+  // Set the type of filter (artist, fair, etc.) for custom behavior
+  type?: string
```

Comment:
Can we add an union here of possible options?

Reply:
That might be built out later, for now can restrict this prop to just "fair"?

Basically I need a prop to control the display of the filters (filter is a shared component, but some behavior re: the display of a particular filter is custom, basically the same kind of silliness as yesterday's `showTopBorder` discussion. The perils of sharing w/ custom behavior!).

Reply:
Yup, if there's only one option for now we should restrict it to just that one!

Reply:
One q tho would be -- is there a better prop name for the behavior this is enabling? Since it shows `FollowedArtistsFilter` -- maybe, `showFollowedArtistsFilter` or some such thing? It seems like that could be useful more generally.

Reply:
Thinking further... a better option would be to add a property called `filters` that takes an array of filters from https://github.com/artsy/force/blob/7a3946a46bc12db55770e272c5ca18a881d5213d/src/v2/Components/v2/ArtworkFilter/ArtworkFilters/index.tsx.

So like 
```
filters={[FollowedArtistsFilter, MediumFilter, PriceRangeFilter, ...]}
```
and then in that `ArtworkFilters` file, check to see if `filters` exists in the context and then do something like 
```
filters.map(Filter => <Filter />)
```
the filters prop can have a default argument that renders out the common list.

Reply:
Interesting!

I basically came up with the prop due to not wanting something analogous to `showTopBorder`, aka `showFollowedArtistsFilter`.

Thinking a bit about the custom behavior that I've heard discussed around filtering (mostly sorting), and it seems as though every 'instance' of a filter _could_ be custom. Frequently this is based on the page type it appears on (filter on the artist page behaves like X for instance vs. on another page), but, at least with collections, might even vary based on the type of collection it is (even though the page type is the same: 'collection').

I'm liking your suggestion though. It basically takes the specific ask (custom filter display), and makes that generic via `filters`, which seems reasonable. And it doesn't add any overhead of the more specific `showFollowedArtists` or the too-catch-all `type`.

Reply:
Nice, yah if that prop accepts JSX / component that would probably be the easiest / most flexible through time. And if no `filters` prop provided, it falls back to the component thats already being used (via default arg). (I also worry a bit about the typings if we were to go the array route... could get weird)

Reply:
Yea that is way nicer.